tests/conftest.py

```diff
@@ -875,3 +875,11 @@ def duthost_console(localhost, creds, request):
                        console_password=creds['console_password'][vars['console_type']])
     yield host
     host.disconnect()
+
+@pytest.fixture(scope='session', autouse=True)
+def cleanup_cache_for_session(request):
+    yield
+    tbname, tbinfo = get_tbinfo(request)
+    cache.cleanup(zone=tbname)
+    for a_dut in tbinfo['duts']:
+        cache.cleanup(zone=a_dut)
```

Comment:
Curious why cleanup happens after test instead of before test? If the cache is stale, won't it fail the test? If a test particularly changed something in the cache and don't want to keep the change for next test, then it should clean up after test, but in that case, the fixture shouldn't be autouse, each test should decide to use the fixture or not.

Reply:
Removed the cleanup_cache fixture as this cleanup is being added to test_pretest in PR #2978